Ghost Host/css and images/css.css

```diff
@@ -0,0 +1,30 @@
+@font-face {
```

Comment:
Please, name your files accrording to their functions. It's quite hard to guess what is css.css for.